elixir.nanorc

```diff
@@ -44,7 +44,7 @@ color brightcyan "(^|[[:space:]])##.*$"
 color green start="\"\"\"" end="\"\"\""
 
 ## Some common markers
-color brightcyan "(XXX|TODO|FIXME|\?\?\?)"
+color ,cyan "(XXX|TODO|FIXME|\?\?\?)"
```

Comment:
I changed foreground cyan to background, because: 1) it is more noticeable, and 2) to align with how existing TODOs are being highlighted.